src/router.ts

```diff
@@ -168,12 +168,19 @@ class Router {
   // Reads all DIP Deposits and decides whether to send it to the mm_callback
   async checkMMPrices(): Promise<void> {
     let openPositionCount = 0;
-    for (const dipDeposit of Object.values(this.dips)) {
-      if (dipDeposit.qtyTokens > 0) {
-        openPositionCount++;
-        await this.route(dipDeposit);
+    try {
```

Comment:
Are you sure this would catch the API response 'Internal server error'? We still get a response from the API, we just don't like it. My understanding this would catch an actual error running route() which we haven't seen yet or at least wasn't this specific edge case